### PR TITLE
Bump to libocr master 2271752fa829

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

**Review by aalu1418 — COMMENTED** [review]
✅ 

but waiting for approval from other folks because I made changes to the PR

**Review by smickovskid — APPROVED** [review]
LGTM

**Review by aalu1418 — APPROVED** [review]

**Pull request merged**
